index.js

```diff
@@ -306,14 +307,22 @@ class TuyaDevice extends EventEmitter {
     const timeStamp = parseInt(new Date() / 1000, 10);
 
     // Construct payload
-    const payload = {
-      devId: options.devId || this.device.id,
-      gwId: this.device.gwID,
-      uid: '',
+    let payload = {
       t: timeStamp,
       dps
     };
 
+    if (options.cid) {
+      payload.cid = options.cid;
```

Comment:
Again, docs need to be updated for this parameter.

Reply:
Done!